docs/dac/cue/dashboard.md

```diff
@@ -0,0 +1,84 @@
+# Dashboard builder
+
+The Dashboard builder helps creating dashboards in the format expected by Perses.
+
+## Usage
+
+```cue
+package myDaC
+
+import (
+	dashboardBuilder "github.com/perses/perses/cue/dac-utils/dashboard"
+)
+
+dashboardBuilder & {} // input parameters expected
+```
+
+## Parameters
+
+| Parameter      | Type                                                                                                                                   | Description                                                                                                         |
+|----------------|----------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------|
+| `#name`        | string                                                                                                                                 | The name of the dashboard.                                                                                          |
+| `#project`     | string                                                                                                                                 | The project to which the dashboard belongs.                                                                         |
+| `#variables`   | [...[VariableSpec](../../api/variable.md#dashboard-level)]                                                                             | An array of variables defined for the dashboard.                                                                    |
+| `#panelGroups` | map[string]: { layout: [Layout](../../api/dashboard.md#layoutspec), panels: map[string]: [Panel](../../api/dashboard.md#panelspec) } | A map where each key is a panel group name, and the value is an object containing layout and panels for that group. |
```

Comment:
such links with section refs that don't contain any underscores or dashes actually don't work properly when checking the rendered doc on github (the automatic scrolling isnt happening), but that's currently how mdox wants the links to be. I'm going to address this in a follow-up PR.